app/views/admin_users/sessions/new.html.erb

```diff
@@ -0,0 +1,26 @@
+<h2>Log in as Admin</h2>
+
+<%= form_for(resource, as: resource_name, url: session_path(resource_name)) do |f| %>
```

Comment:
jak nie zmieniasz widokow to nie wiem czy bym je generowal. Nie zasmiecaja troche?

Reply:
No właśnie tak myślałem czy to nie jest zbędne. W sumie ten sam efekt udało mi się uzyskać w 5 liniach kodu, a przy okazji nauczyłem się nowego helpera. :)